testsuite/MDAnalysisTests/topology/test_fetch_pdb.py

```diff
@@ -27,9 +27,12 @@
 from MDAnalysis.topology.PDBParser import HAS_POOCH
 
 from urllib import request
+from shutil import rmtree
+from pathlib import Path
 
 if HAS_POOCH:
     from requests.exceptions import HTTPError
+    from pooch import os_cache
```

Comment:
I'd avoid importing into the namespace. When reading code below it's not clear where os_cache is from. It's much clearer to just `import pooch` and then later use `pooch.os_cache`.